### ISSUE TITLE
Error building for Android - package android.support.annotation does not exist

### ISSUE THREAD

**Issue by ofiron01** [react-native-community/react-native-masked-view#25]
# Bug

Compilation error on version 0.1.1
node_modules/@react-native-community/masked-view/android/src/main/java/org/reactnative/maskedview/RNCMaskedViewManager.java:3: error: package android.support.annotation does not exist
import android.support.annotation.Nullable;
                                 ^
1 error

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':@react-native-community_masked-view:compileDebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

## Environment info

React native info output:

```bash
info Fetching system and libraries information...
System:
    OS: macOS 10.15
    CPU: (12) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
    Memory: 330.52 MB / 16.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 10.15.3 - /usr/local/bin/node
    Yarn: 1.16.0 - /usr/local/bin/yarn
    npm: 6.4.1 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.0, DriverKit 19.0, macOS 10.15, tvOS 13.0, watchOS 6.0
    Android SDK:
      API Levels: 23, 26, 28
      Build Tools: 23.0.1, 26.0.2, 28.0.3
      System Images: android-26 | Google APIs Intel x86 Atom, android-28 | Google APIs Intel x86 Atom, android-28 | Google Play Intel x86 Atom
  IDEs:
    Xcode: 11.0/11A420a - /usr/bin/xcodebuild
  npmPackages:
    react-native: 0.61.2 => 0.61.2 
  npmGlobalPackages:
    react-native-cli: 2.0.1
    react-native-git-upgrade: 0.2.7
```

Library version: 0.1.1

## Steps To Reproduce

1. Build for Android
...

Describe what you expected to happen:

1. Use the library without issues

**Comment by leongxj** [comment]
you may check this out

https://stackoverflow.com/questions/40380519/error-package-com-android-annotations-does-not-exist

**Comment by tlapuste** [comment]
Could someone please merge in the following pull request which fixes this issue, and push out a new release? This package is broken for Android on react 0.6x in the meanwhile.

https://github.com/react-native-community/react-native-masked-view/pull/21

**Comment by ofiron01** [comment]
Temporary workaround , using jetifier via mom postinstall script @tlapuste

**Comment by FonDorn** [comment]
This package is building successfully on android for initial react native application (`react-native init`) for react native version less and above then 60. On what android device you try to run?

**Comment by FonDorn** [comment]
merged #21
